src/xenia/kernel/user_module.cc

```diff
@@ -804,7 +804,7 @@ void UserModule::Dump() {
     sb.AppendFormat("\n");
   }
 
-  xe::LogLine(xe::LogLevel::Info, 'i', sb.GetString());
+  xe::LogLine(xe::LogLevel::Error, 'i', sb.GetString());
```

Comment:
I don't think we want to output all module information like this. It would be preferable to log the title ID of the main module once it's available, before any module dumps occur.